backends/ebpf/psa/README.md

```diff
@@ -141,7 +141,17 @@ An inner map is a hash map storing clone session/multicast group members, accord
 `struct list_key_t` (BPF map key) and `struct element` (value). To add a new clone session/mutlicast group, a control plane must add
 a new element to the outer map (indexed by clone session or multicast group identifier referenced by `clone_session_id` 
 or `multicast_group` in a PSA program) and initialize an inner map. To add a new clone session/multicast group member,
-a control plane must add new element to the inner map.
+a con1trol plane must add new element to the inner map.
+
+## PSA externs
+
+### Digest
+
+Digests are intended to carry a small piece of user-defined data from the data plane to a control plane.
+The PSA-eBPF compiler translates each Digest instance into `BPF_MAP_TYPE_QUEUE` that implements a FIFO queue.
+If a deparser triggers the `pack()` method, an eBPF program inserts data defined for a Digest into the BPF queue map using `bpf_map_push_elem`.
+A user space application is responsible for performing periodic queries to this map to read a Digest message. It can use either
+`psabpf-ctl digest get pipe` or `bpf_map_lookup_and_delete_elem` C API.
```

Comment:
`bpf_map_lookup_and_delete_elem` comes from libbpf, might be worth to mention C API from our `psabpf`